app/src/main/res/layout/activity_ask_for_inspection.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context="in.avimarine.boatangels.activities.AskInspectionActivity">
+
+  <TextView
+    android:id="@+id/ask_for_inspection_title"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/ask_for_inspection_title"
+    android:textColor="#000000"
+    android:textSize="24dp"
```

Comment:
textSize should use sp and not dp:
https://stackoverflow.com/questions/23981260/should-use-sp-instead-of-dp-for-text-sizes